main.py

```diff
@@ -131,6 +138,33 @@ def get_dice_to_save(self):
         num_arr = map(int, nums.split(" "))
         return [num for num in num_arr if num > len(self.saved_dice)]
 
+    def get_dice_to_save2(self, rolled):
+        """
+        Accepts `1 2 3` or `1, 2, 3`
+        """
+        roll_orig = rolled
+        nums = input("Which dice do you want to save? ")
+        if nums is None or len(nums) <= 0:
+            return []
+        nums = list(map(
+            int, "".join(list(filter(lambda c: c in '0123456790 ',
+                                     nums))).split(" ")))
+        if len(nums) > 3 or len(nums) + len(self.saved_dice) > 5:
```

Comment:
Why the restriction on len(nums) > 3?

Reply:
Whoops I inverted the rule.
It should be “you can only reroll 3”

Reply:
Couldn't it be up to 5?